packages/manager/modules/iam/src/components/advancedModeSwitch/advancedModeSwitch.controller.js

```diff
@@ -0,0 +1,26 @@
+export default class IamAdvancedModeSwitchController {
+  /* @ngInject */
+  constructor($q, $rootScope, PreferencesService) {
+    this.$q = $q;
+    this.$rootScope = $rootScope;
+    this.PreferencesService = PreferencesService;
+  }
+
+  onChangeMode(modelValue) {
+    return this.$q
+      .resolve(
```

Comment:
Same as https://github.com/ovh/manager/pull/9146#discussion_r1169591502

Reply:
It's applied

Reply:
Can you do a second check?

Reply:
It's applied